### ISSUE TITLE
Final Submission/Keyword not accessible

### ISSUE THREAD

**Issue by cam156** [psu-stewardship/etda#127]
not reading dialog for deleting keyword
backspace starts to delete which is expected behavior
entering in a space does not allow tabbing out of the field because it’s starting to create a new keyword
cannot tab out of keyword
